packages.yml

```diff
@@ -1,3 +1,3 @@
 packages:
  - package: fishtown-analytics/dbt_utils
-   version: '>=0.1.20'
\ No newline at end of file
+   version: ['>=0.1.20', "<0.5.0"]
```

Comment:
If you want, this can be increased again to "<0.6.0". Basically I messed some stuff up and utils releases are a trash fire right now :D

Reply:
💀 💀 I'm dead Just fixed!